asv_bench/benchmarks/algorithms.py

```diff
@@ -113,4 +132,21 @@ def time_series_dates(self, df):
         hashing.hash_pandas_object(df['dates'])
 
 
+class Quantile(object):
```

Comment:
can remove the Match one; was remove in 0.23.0 i think.

Reply:
Done